config/initializers/fog_uploader.rb

```diff
@@ -0,0 +1,8 @@
+CarrierWave.configure do |config|
+    config.fog_credentials = {
+        provider:                         'Google',
+        google_storage_access_key_id: ENV.fetch("GOOGLE_ACCESS_KEY"),
+        google_storage_secret_access_key: ENV.fetch("GOOGLE_SECRET_ACCESS_KEY")
+      }
+      config.fog_directory = 'dischords-development'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.